sdk/confidentialledger/azure-confidentialledger/CHANGELOG.md

```diff
@@ -1,12 +1,23 @@
 # Release History
 
-## 1.0.0b2 (Unreleased)
+## 1.0.0 (Unreleased)
+
+GA Data Plane Python SDK for Confidential Ledger.
 
 ### Bugs Fixed
+- Fixes [[confidentialledger] mypy: function has duplicate type signatures (#23356)](https://github.com/Azure/azure-sdk-for-python/issues/23356)
+- User ids that are certificate fingerprints are no longer URL-encoded in the request URI.
+
+### Breaking Changes
+- Removed all models. Methods now return JSON directly as a side-effect of migrating to DPG generation.
+- In accordance with the updated Swagger, the `sub_ledger_id` fields are now named `collection_id`.
```

Comment:
nit: no need to mention swagger, you can just say the params have been renamed. End users don't have to know what swagger is